src/Storages/MergeTree/ParallelReplicasReadingCoordinator.cpp

```diff
@@ -1132,6 +1146,7 @@ void ParallelReplicasReadingCoordinator::initialize(CoordinationMode mode)
 
 ParallelReplicasReadingCoordinator::ParallelReplicasReadingCoordinator(size_t replicas_count_) : replicas_count(replicas_count_)
 {
+    LOG_DEBUG(getLogger("ParallelReplicasReadingCoordinator"), "replicas_count={}", replicas_count);
```

Comment:
Should be more verbose (otherwise at least 00002_log_and_exception_messages_formatting will complain)